Week2/todo_app/program.js

```diff
@@ -62,11 +123,121 @@ dbConnection.connect(function(err) {
     console.log('connected as id ' + dbConnection.threadId);
 
     const todoModel = new TodoModel(dbConnection);
-    todoModel.load(function(err, todoItems) {
-        if(err) {
+    todoModel.load(function (err, todoItems) {
+        if (err) {
             console.log("error loading TODO items:", err);
         }
 
         console.log("existing todo items:", todoItems);
     });
-});
\ No newline at end of file
+
+    case 'delete':
+        {
+            todoModel.delete(arg1, function (err) {
+                if (err) {
+                    console.log("error deleting TODO item:", err);
+                }
+                console.log('TODO item:', arg1, 'has been deleted!')
+            });
+            break;
+        }
+
+    case 'tag':
+        {
+            todoModel.tagTodoItem(arg1, arg2, function (err) {
+                if (err) {
+                    console.log("error tagging TODO item:", err);
+                }
+                console.log('TODO item:', arg1, 'has been tagged to', arg2)
```

Comment:
Did you try your loggings here? Does it look like what you have in mind? Remember that you get the queryResult as a json object and you need some extra work to fit it within a sentence (as how you plan to log here)